src/action/platform_info.rs

```diff
@@ -0,0 +1,274 @@
+// Copyright 2020 Google LLC
+//
+// Use of this source code is governed by an MIT-style license that can be found
+// in the LICENSE file or at https://opensource.org/licenses/MIT.
+
+extern crate sys_info;
+
+/// Using for `uname` syscall.
+extern crate libc;
+
+use sys_info::{linux_os_release, os_type, hostname};
+use libc::{uname, utsname, c_char};
+use crate::session::{self, Session};
+use std::ffi::CStr;
+use std::option::Option;
+use std::fmt::{Display, Formatter};
+
+use rrg_proto::Uname;
+
+#[derive(Debug)]
+enum Error {
+    CannotGetOSType(sys_info::Error),
+    CannotGetLinuxRelease(sys_info::Error)
+}
+
+impl std::error::Error for Error {
+    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
```

Comment:
I tend to put a blank line after `impl`, `trait` and the first item (it looks better for some reason for me).

Reply:
Ok, I fixed that